### DISCUSSION TITLE
POST broadcasts instead of POST notifications

### DISCUSSION THREAD

**Discussion by ross-kortx** [magicbell/community#221] [Questions] [answered]
Hi, when I setup MagicBell integration, the documentation for the REST API was to use POST notifications to create notifications.
I noticed recently that the endpoint now POST broadcasts, but my original requests still work fine.

Is the old endpoint going to stop working at some point?

**Comment by unamashana** [comment, marked as answer]
Hey @ross-kortx , there are no plans to deprecate them now, but we won't be adding any new features to these endpoints. Here is a migration guide if you'd like to migrate. The migration is pretty quick for most of our customers and offers better rate limits and support for new features - https://www.magicbell.com/docs/guides/migrations/migrating-to-post-broadcast
